packages/ui-stencil/src/components/internal/orama-sliding-panel/orama-sliding-panel.tsx

```diff
@@ -67,23 +67,26 @@ export class SlideInPanel {
 
   render() {
     return (
-      <div class={{ 'slide-container': true, 'slide-up': this.isOpen }} tabIndex={this.isOpen ? 0 : -1}>
-        <div class="slide-container-header">
-          <button
-            onClick={() => this.closePanel()}
-            aria-expanded={this.isOpen ? 'true' : 'false'}
-            aria-label="Close panel"
-            aria-controls="panel"
-            class="close-button"
-            type="button"
-          >
-            <ph-x size="18" />
-          </button>
+      <Fragment>
+        <div class={{ 'slide-container': true, 'slide-up': this.isOpen }} tabIndex={this.isOpen ? 0 : -1}>
+          <div class="slide-container-header">
+            <button
+              onClick={() => this.closePanel()}
+              aria-expanded={this.isOpen ? 'true' : 'false'}
+              aria-label="Close panel"
+              aria-controls="panel"
+              class="close-button"
+              type="button"
+            >
+              <ph-x size="18" />
+            </button>
+          </div>
+          <div id="panel" role="region" aria-hidden={!this.isOpen} tabindex="-1" class="slide-container-inner">
+            <slot />
+          </div>
         </div>
-        <div id="panel" role="region" aria-hidden={!this.isOpen} tabindex="-1" class="slide-container-inner">
-          <slot />
-        </div>
-      </div>
+        <div class={{ 'slide-backdrop': true, visible: this.isOpen }} />
```

Comment:
It would be nice to have the ability to add a backdrop using props. Since this is a sliding panel component, there may be situations where the backdrop is more effective, and others where we don't need it at all. In this case, we can include the backdrop only in the 'embedded' layout